python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -232,3 +240,61 @@ def predict_proba(self, dataset: pd.DataFrame) -> np.ndarray:
         """
         predictions = self._mlflow_predict(dataset)
         return np.stack(predictions.probs.values)
+
+
+class WrappedObjectDetectionModel:
```

Comment:
this looks great!  can you please add tests to the new wrapper and perhaps even add some documentation on the predict function specification for object detection?  maybe the docs can be added in a separate PR and we can strategize on where to put them in the repo.  I imagine we would want to put them in some place like:
https://github.com/microsoft/ml-wrappers/tree/main/docs

Reply:
Working on the tests now. As for documentation, would I add it here? https://github.com/microsoft/ml-wrappers/blob/main/docs/WrapperSpecifications.md Thanks!

Reply:
yes we can add automl support + tests in a separate PR, I am guessing more changes might be needed to support their models?